spec/component8/verifyCredentials.js

```diff
@@ -0,0 +1,18 @@
+module.exports = verify;
+
+function sleep(ms) {
+    return new Promise(resolve => setTimeout(resolve, ms));
+}
+
+async function verify(credentials, cb) {
+    try {
+        await sleep(100);
+        const q = true;
+        if (q) {throw new Error('Verification failed :(');}
```

Comment:
Too complicated `if (true)` logic
Also, please check syntax of this file

Reply:
Ha, I forgot about this 😄 was just changing different things to test, sorry